traffic_ops/traffic_ops_golang/traffic_ops_golang.go

```diff
@@ -68,6 +69,29 @@ func main() {
 		fmt.Println(strings.Join(plugin.List(), "\n"))
 		os.Exit(0)
 	}
+	if *showRoutes {
+		fake := routing.ServerData{Config: config.NewFakeConfig()}
+		routes, _, _, _ := routing.Routes(fake)
+		if len(*configFileName) != 0 {
+			cfg, err := config.LoadCdnConfig(*configFileName)
+			if err != nil {
+				fmt.Printf("Loading cdn config from '%s': %v", *configFileName, err)
+				os.Exit(1)
+			}
+			perlRoutes := routing.GetRouteIDMap(cfg.PerlRoutes)
+			disabledRoutes := routing.GetRouteIDMap(cfg.DisabledRoutes)
+			for _, r := range routes {
+				_, isBypassedToPerl := perlRoutes[r.ID]
+				_, isDisabled := disabledRoutes[r.ID]
+				fmt.Printf("id=%d\tmethod=%s\tversion=%.1f\tpath=%s\tcan_bypass_to_perl=%t\tis_bypassed_to_perl=%t\tis_disabled=%t\n", r.ID, r.Method, r.Version, r.Path, r.CanBypassToPerl, isBypassedToPerl, isDisabled)
+			}
+		} else {
+			for _, r := range routes {
+				fmt.Printf("id=%d\tmethod=%s\tversion=%.1f\tpath=%s\tcan_bypass_to_perl=%t\n", r.ID, r.Method, r.Version, r.Path, r.CanBypassToPerl)
```

Comment:
Why not just `fmt.Println(r)` since you made the `String` method?

Reply:
the `String` method just uses a single space as a separator, whereas this uses a tab. I think the tab-delimited output looks more readable in the output. I also wanted it to basically match the `Printf` just above this.

Reply:
No tabs are definitely the way to go for tabular data like this. Just missed that the `String` method used spaces.